frodo.js

```diff
@@ -1,51 +1,50 @@
 /**
```

Comment:
SCRIPT-Consider using more descriptive naming for this boolean variable to clarify its purpose.